src/jenesis/cmd/add/deployment.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import os
+import toml
+
+from jenesis.config import Config
+
+
+def run(args: argparse.Namespace):
+
+    project_root = os.path.abspath(os.getcwd())
+    contract_root = os.path.join(project_root, "contracts", args.contract)
+
+    # check that we are actually running the command from the project root
+    if not os.path.exists(os.path.join(project_root, "jenesis.toml")):
+        # pylint: disable=all
+        print("Please run command from project root")
+        return
+
+    # check if the contract exists
+    if not os.path.exists(contract_root):
+        print(f'Contract "{args.contract}" doesnt exist')
+        return
+
+    data = toml.load("jenesis.toml")
+
+    cfg = Config.load(project_root)
+    profiles = list(cfg.profiles.keys())
+
+    for profile in profiles:
+        contract_data = data["profile"][profile]["contracts"][args.contract].copy()
```

Comment:
It shouldn't really happen, but just in case someone does not have this contract in one of their profiles, I guess we should just skip it instead of getting a `KeyError` here.

Reply:
ok